suite-native/navigation/src/components/NavigationContainerWithAnalytics.tsx

```diff
@@ -59,6 +60,16 @@ export const NavigationContainerWithAnalytics = ({ children }: { children: React
                     currentScreen: currentRouteName,
                 },
             });
+
+            Sentry.addBreadcrumb({
```

Comment:
This is nice, but it can be useful to see the current route as sentry tag so issues/events can be filtered by the route. See https://github.com/trezor/trezor-suite/blob/develop/packages/suite/src/middlewares/suite/sentryMiddleware.ts/#L120

Of course it can be done separately, but I believe it's a low hanging fruit.

Reply:
added in 60cd58fb1f

Reply:
added in 60cd58fb1f